tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -103,6 +103,7 @@ const onlyOfficeCanvasEditorSelector = '#id_viewer_overlay'
 const onlyOfficeCanvasCursorSelector = '#id_target_cursor'
 const collaboraCanvasEditorSelector = '.leaflet-layer'
 const textEditorTextArea = '#text-editor-input'
+const filesActionDropdown = `//div[contains(@class, 'tippy-box')]//button[contains(@class, 'oc-files-actions-%s-trigger')]`
```

Comment:
we have this selector `filesAction` in line 40. Will it work?

Reply:
ah, that's for batch action button

Reply:
`filesAction` has same class name and resolves into two elements.